crates/rolldown_plugin/src/pluginable.rs

```diff
@@ -458,4 +460,8 @@ impl<T: Plugin> Pluginable for T {
   fn call_transform_ast_meta(&self) -> Option<PluginHookMeta> {
     Plugin::transform_ast_meta(self)
   }
+
+  fn hook_usage(&self) -> HookUsage {
```

Comment:
Some background:
- Methods of `Pluginable` should be indentical as `Plugin`'s. But to avoid naming conflicts, I just simply add `call_` prefix to each method.
- I don't think we need to follow it. Just add some contexts here.

Reply:
Ok, I will rename it to `call_hook_usage`